### DISCUSSION TITLE
2005/09/24/%E4%BF%A1%E5%8F%B7%E8%B0%83%E7%90%86%E6%A8%A1%E5%9D%97%E7%9A%84%E5%8A%9F%E8%83%BD

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#49] [Announcements]
# 2005/09/24/%E4%BF%A1%E5%8F%B7%E8%B0%83%E7%90%86%E6%A8%A1%E5%9D%97%E7%9A%84%E5%8A%9F%E8%83%BD

阮奇桢

https://blog.qizhen.xyz/2005/09/24/%E4%BF%A1%E5%8F%B7%E8%B0%83%E7%90%86%E6%A8%A1%E5%9D%97%E7%9A%84%E5%8A%9F%E8%83%BD

**Comment by ruanqizhen** [comment]
楠
2007年06月11日 01:19 编辑
（按错键了~）接前——影响？有啥解决办法没？ 期待解答，非常感谢！

回复
楠
2007年06月11日 01:16 编辑
哇塞~长知识~！想请问，在工厂现场调试时，我采集的是扭矩传感器发出的数字信号通过 自制的光隔，用示波器 显示信号良好，但信号进入工控机通过PCI6221采集卡后在Labview程序中显示的频率波形却有高于采样率两倍的噪声~ 是因为PCI6221这块卡本身有缺陷吗，在工控机内会受

回复
miu
2006年04月23日 20:39 编辑
信号调理模块可以用在路由器上面吗?很着急的问题,可以尽快给我回复吗?
ADI的7B34系列与上文描述的是否起到一样的作用呢?

回复
